src/screens/Recovery/Lightning.tsx

```diff
@@ -0,0 +1,304 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import { ActivityIndicator, StyleSheet, View } from 'react-native';
+import { useTranslation } from 'react-i18next';
+
+import lm, { ldk, TLdkData } from '@synonymdev/react-native-ldk';
+import { View as ThemedView } from '../../styles/components';
+import List, { EItemType, IListData, ItemData } from '../../components/List';
+import { Text01S } from '../../styles/text';
+
+import NavigationHeader from '../../components/NavigationHeader';
+import SafeAreaInset from '../../components/SafeAreaInset';
+import Button from '../../components/Button';
+import { RecoveryStackScreenProps } from '../../navigation/types';
+import { useSelectedSlashtag } from '../../hooks/slashtags';
+import { SlashtagsProvider } from '../../components/SlashtagsProvider';
+import {
+	EBackupCategories,
+	fetchBackup,
+	listBackups,
+} from '../../utils/backup/backpack';
+import { EAvailableNetworks } from '../../utils/networks';
+import Dialog from '../../components/Dialog';
+import { startWalletServices } from '../../utils/startup';
+import { showToast } from '../../utils/notifications';
+import RNExitApp from 'react-native-exit-app';
+import { selectedNetworkSelector } from '../../store/reselect/wallet';
+import { useSelector } from 'react-redux';
+import { bytesToString } from '../../utils/converters';
+import { setLdkStoragePath } from '../../utils/lightning';
+import { TAccountBackup } from '../../store/types/backup';
+const Lightning = (
+	props: RecoveryStackScreenProps<'Lightning'>,
+): ReactElement => {
+	return (
+		<SlashtagsProvider>
+			<LightningWithSlashtags {...props} />
+		</SlashtagsProvider>
+	);
+};
+
+const LightningWithSlashtags = ({
+	navigation,
+}: RecoveryStackScreenProps<'Lightning'>): ReactElement => {
+	const { t } = useTranslation('security');
+	const slashtag = useSelectedSlashtag();
+	const [history, setHistory] = useState<IListData>({
+		title: 'Loading backups...',
+		data: [],
+	});
+	const [showConfirmRecoveryDialog, setShowConfirmRecoveryDialog] =
+		useState(false);
+	const [isFetchingBackup, setIsFetchingBackup] = useState(false);
+	const [backup, setBackup] = useState<TAccountBackup<TLdkData> | null>(null);
+	const [isRecoveringChannels, setIsRecoveringChannels] = useState(false);
+	const [recoveredSats, setRecoveredSats] = useState(0);
+	const [showLdkRecoverySuccessDialog, setShowLdkRecoverySuccessDialog] =
+		useState(false);
+	const selectedNetwork = useSelector(selectedNetworkSelector);
+
+	//On mount
+	useEffect(() => {
+		if (!slashtag || history.data.length > 0) {
+			return;
+		}
+
+		const listLdkBackups = async (): Promise<void> => {
+			const res = await listBackups(
+				slashtag.slashtag,
+				EBackupCategories.ldkComplete,
+				EAvailableNetworks.bitcoin,
```

Comment:
```suggestion
				__DEV__ ? selectedNetwork : EAvailableNetworks.bitcoin,
```

Make dev testing on regtest easier.